react/ProductSummaryList.tsx

```diff
@@ -143,17 +143,19 @@ function ProductSummaryList(props: PropsWithChildren<Props>) {
   })
 
   const { products } = data ?? {}
+  // Not using ?? operator because listName can be ''
+  // eslint-disable-next-line no-unneeded-ternary
+  const listName = rawListName ? rawListName : 'List of products'
 
   const productClick = useCallback(
-    (product: any) => {
+    (product: any, position: number) => {
```

Comment:
I'm not sure, but if there is something new to add in the future to this function will need to add a 3rd parameter, what do you think about using an object as a second parameter?

Reply:
Agree on this one because that's a public API that can evolve and we should avoid breaking changes